tests/runtime_neutral/test_governed_evolution_smoke.ps1

```
@@ -0,0 +1,213 @@
+        'atomic_skill_call_chain',
+        'proposal_layer',
+        'proposal_layer_markdown',
+        'warning_cards',
+        'preflight_checklist',
+        'remediation_notes',
+        'candidate_composite_skill_draft',
+        'threshold_policy_suggestion',
+        'application_readiness_report',
+        'application_readiness_markdown'
+    )) {
+        Require (Has-Property $artifacts $artifactName) "runtime-summary.json missing artifacts.$artifactName" $failures
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Require' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)